backend/app/controllers/commentController.ts

```diff
@@ -117,7 +116,7 @@ class CommentController {
     if (!post || !profile)
       return res
         .status(422)
-        .json({ message: 'Invalid post or profile given. Likely an attempt to comment on "new-post" page.' })
+        .json({ message: 'Invalid post or profile given. Likely an attempt to comment on "new-post" page' })
```

Comment:
```suggestion
        .json({ message: 'Invalid post or profile given. Likely an attempt to comment on \'new-post\' page' })
```

Citations are single-quote

Reply:
Wrong backend